### PR TITLE
Fixed reading properties of null value

### LINKED ISSUE #49: Fix the issue with null values
Reference:
- https://github.com/bigbinary/neeto-form-web/issues/2351

### PR DESCRIPTION
**Pull request by Thejus-Paul**
Fixes #49

### CODE REVIEW

**Comment by Thejus-Paul** [comment]
@amaldinesh7 _a Please review.

File: src/components/index.js [hunk 1]
```diff
@@ -79,8 +80,8 @@ const TeamMembers = ({
 
   const SubHeaderLeftActionBlock = () => (
     <Typography style="h4" component="h4">
-      {filteredMembers.length}{" "}
-      {filteredMembers.length === 1 ? metaName : `${metaName}s`}
+      {!isEmpty(filteredMembers) && filteredMembers?.length}{" "}
```
Comment (amaldinesh7) [inline src/components/index.js:83]: Try to combine these under a single condition check of `!isEmpty(filteredMembers)`

File: src/components/index.js [hunk 2]
```diff
@@ -187,7 +188,7 @@ const TeamMembers = ({
             searchProps={searchProps}
             actionBlock={HeaderActionBlock(metaName)}
           />
-          {filteredMembers.length ? (
+          {!isEmpty(filteredMembers) && filteredMembers?.length ? (
```
Comment (amaldinesh7) [inline src/components/index.js:191]: Please do create a separate variable for this condition `!isEmpty(filteredMembers)` and use it.

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@Thejus-Paul _a Please check the comments.

**Comment by Thejus-Paul** [comment]
@amaldinesh7 _a Please review.

File: src/components/index.js [hunk 3]
```diff
@@ -79,8 +80,11 @@ const TeamMembers = ({
 
   const SubHeaderLeftActionBlock = () => (
     <Typography style="h4" component="h4">
-      {filteredMembers.length}{" "}
-      {filteredMembers.length === 1 ? metaName : `${metaName}s`}
+      {`${getMembersCount(filteredMembers)} ${
```
Comment (amaldinesh7) [inline src/components/index.js:83]: You are invoking the same function with the same parameter 3 times. please do assign the value to a variable and use it.

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@Thejus-Paul _a Please check the comment.

**Comment by Thejus-Paul** [comment]
@amaldinesh7 _a Please review.

**Review by amaldinesh7 — APPROVED** [review]

**Pull request merged**